### DISCUSSION TITLE
Using the Addon

### DISCUSSION THREAD

**Discussion by Kesera-04** [kheina-com/Blue-Blocker#199] [Q&A]
This is kind of embarrassing, but I can't actually figure out how to get the addon to start blocking people. I've selected my settings but I'm unsure on how actually to run it.

**Comment by bkh555** [comment]
You need to be logged into twitter, "Pause user collection" needs to be unchecked, and you have to have a blue check post loaded on your screen. I think the only feed it doesn't pick up from is the likes tab.
